blockchain-workbench/scripts/upgrade/azureBlockchainWorkbenchUpgradeToLatest.ps1

```diff
@@ -24,24 +24,25 @@ None. You cannot pipe objects to this script.
 None. This script does not generate any output.
 .EXAMPLE
 
-C:\tmp> .\azureBlockchainWorkbenchUpgradeTov1_8_0.ps1 -SubscriptionID "<subscription_id>" -ResourceGroupName "<workbench-resource-group-name>"
+C:\tmp> .\azureBlockchainWorkbenchUpgradeTov1_8_.ps1 -SubscriptionID "<subscription_id>" -ResourceGroupName "<workbench-resource-group-name>"

```

Comment:
Should probably be azureBlockchainWorkbenchUpgradeTov1_8_1?

Reply:
Fixed